cli_helpers/tabular_output/sql_output_adapter.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+
+supported_formats = (
+    "sql-insert",
+    "sql-update",
+    "sql-update-1",
+    "sql-update-2",
+)
+
+preprocessors = ()
+
+
+def escape_for_sql_statement(value):
+    if isinstance(value, bytes):
+        return f"X'{value.hex()}'"
+    else:
+        return "'{}'".format(value)
+
+
+def adapter(data, headers, table_format=None, **kwargs):
+    """
+    This function registers supported_formats to default TabularOutputFormatter
+
+    Parameters:
+        data: query result
+        headers: columns
+        table_format: values from supported_formats
+        kwargs:
+            extract_tables: extract_tables function. For example, in pgcli.packages.parseutils.tables there is a function extract_tables
+            delimiter: Character surrounds table name or column name when it conflicts with sql keywords.
+                       For example, mysql uses ` and postgres uses "
+    """
+    extract_table_func = kwargs.get("extract_tables")
```

Comment:
This is one way to do it. Alternatively, the CLI could pass in `tables` as part of `kwargs`.

Reply:
I'll try it